crates/web-sys/webidls/enabled/CanvasRenderingContext2D.webidl

```diff
@@ -151,20 +149,18 @@ interface CanvasTransform {
 };
 
 [NoInterfaceObject]
-interface CanvasCompositing {
+interface mixin CanvasCompositing {
   attribute unrestricted double globalAlpha; // (default 1.0)
   [Throws]
   attribute DOMString globalCompositeOperation; // (default source-over)
 };
 
-[NoInterfaceObject]
```

Comment:
Is this a transformation/interpretation that we should bake directly into the webidl parsing? Basically interpreting any interface tagged as `[NoInterfaceObject]` as a mixin instead?